Library/Homebrew/test/.rubocop_todo.yml

```diff
@@ -22,7 +22,7 @@ RSpec/InstanceVariable:
     - 'utils/git_spec.rb'
     - 'version_spec.rb'
 
-# Offense count: 76
+# Offense count: 81
```

Comment:
At this point: let's either not use multiple describes in the test or disable this cop.

Reply:
Which is preferable? It would be relatively easy to just add a single `describe` that encompasses the entire file for each of these. I don't think it would cause much additional clutter at least to the `cmd/*`, `dev-cmd/*`, `rubocops/*` files. I haven't looked at the others.

Alternatively, we can cut down the number of exclusions by using:

```yml
Exclude:
  - 'ENV_spec.rb'
  - 'cleanup_spec.rb'
  - 'cmd/*.rb'
  - 'dependency_spec.rb'
  - 'dev-cmd/*.rb'
  - 'download_strategies_spec.rb'
  - 'exceptions_spec.rb'
  - 'formula_support_spec.rb'
  - 'inreplace_spec.rb'
  - 'language/python_spec.rb'
  - 'options_spec.rb'
  - 'os/mac/mach_spec.rb'
  - 'patch_spec.rb'
  - 'rubocops/*.rb'
  - 'software_spec_spec.rb'
  - 'tap_spec.rb'
  - 'utils/git_spec.rb'
  - 'version_spec.rb'
```

That doesn't really reduce the number of offenses it just cleans up the exclusion list.

Can this go in a separate PR (which I'm happy to do)?

Reply:
> Which is preferable? It would be relatively easy to just add a single `describe` that encompasses the entire file for each of these. I don't think it would cause much additional clutter at least to the `cmd/*`, `dev-cmd/*`, `rubocops/*` files. I haven't looked at the others.

Either is fine with me. Slight preference for "doing what the cop wants".



> Can this go in a separate PR (which I'm happy to do)?

Definitely (and thanks for offering)!

> For now, I'll add new files with only a single `describe` so this isn't increased (although the reason the number jumped so much here is not that I added 5 new offenses, it was just out of date. I only added one)

Perfect 👍🏻